rustfmt.toml

```
@@ -1,3 +1,2 @@
-edition = "2018"
 wrap_comments = true
 imports_granularity = "Crate"
```

nit: suggest newline at end of file 
```suggestion
imports_granularity = "Crate"

```